### ISSUE TITLE
Release to the Mainnet preparation

### ISSUE THREAD

**Issue by Kale98** [IntersectMBO/cc-portal#287]
### Technical requirements

- [x] **Prepare the master branch for push to the Mainnet** (Bloxico CC Portal Team);
- [x] **Pull code from the master branch** (Bloxico DevOps Team);
- [x] **Prepare infrastructure  for frontend, backend, postgres, worker-service, and ipfs-service, db-synch, node** (Intersect)
- [x] **Deploy all necessary services, databases (cc-portal and db_sync)** (Bloxico DevOps Team);
- [x] **Set new variables (worker, backend, aws, ipfs)** (Bloxico DevOps Team based on Bloxico CC Portal Team input)
- [ ] **After the deployment, a script should be entered that will enter Intersect's admin, then the admin should enter the other users, constitution, etc.**

### Comms Intersect

- [ ] @Thomas-Upfield provide a summary in laymans terms of what the CC-Portal is, what it does and who it is for to Intersect comms 2 days before go live.
- [ ] Blog post
- [ ] dev report update

### Comms IOG

- [ ] @Thomas-Upfield to contact IO comms to coordinate with intersect.

[CC-Portal Comms doc](https://docs.google.com/document/d/1Y7_vEEtwyxnsRszg2ZwAyM8hDQ61c1hPR9hWg84qdyI/edit#heading=h.dt6wocnd7c8x) for reference

**Comment by Thomas-Upfield** [comment]
@Kale98 I was unable to add anything directly to your comment above but the following is needed:

**Comms Intersect**

- @Thomas-Upfield provide a summary in laymans terms of what the CC-Portal is, what it does and who it is for to Intersect comms 2 days before go live.
- blog post 
- dev report update

**Comms Input | Output **
- @Thomas-Upfield to contact IO comms to coordinate with intersect.

**Comment by bosko-m** [comment]
@Thomas-Upfield Added items from your comment plus the CC-Portal Comms doc you shared. Lets discuss on the daily standup the exact release date.

**Comment by bosko-m** [comment]
@Kale98 Can you also list the accounts and other config that needs to be set before making it live to the general public under the **Tech reqs** or you can add **App reqs** section for example? Your call. 👍
